src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+public class Deadline extends Task {
+    String deadlineTime;
+    public Deadline(String description, String deadlineTime) {
+        super(description);
+        this.deadlineTime = Parser.parseDate(deadlineTime);
+    }
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + "(" + deadlineTime + "hrs" + ")";
+    }
+    public String saveString() {
```

Comment:
Some might interpret this as a method that saves to storage the info of this object as a string. Can consider `getSaveString()` instead.